conda/base/constants.py

```diff
@@ -270,7 +270,7 @@ class SatSolverChoice(ValueEnum):
 DEFAULT_JSON_REPORTER_BACKEND = "json"
 
 #: The name of the default console reporter backend
-DEFAULT_CONSOLE_REPORTER_BACKEND = "default"
+DEFAULT_CONSOLE_REPORTER_BACKEND = "classic"
```

Comment:
I forgot to change this in the last pull request to `features/reporter-backends`.